File: Calculo-Simples/CalculoSimples.cs
using System;
using System.Linq;

public class CalculoSimples{
    public static void Main(){
        double[] peca1 = Console.ReadLine().Split(' ').Select(double.Parse).ToArray();
        double[] peca2 = Console.ReadLine().Split(' ').Select(double.Parse).ToArray();
        Console.WriteLine($"VALOR A PAGAR: R$ {peca1[1] * peca1[2] + peca2[1] * peca2[2]:0.00}");
    }
}

File: Calculo-Simples/CalculoSimples.java
import java.util.Scanner;

public class CalculoSimples{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        String peca1 = in.nextLine();
        String peca2 = in.nextLine();
        String[] valores1 = peca1.split(" ");
        String[] valores2 = peca2.split(" ");
        System.out.printf("VALOR A PAGAR: R$ %.2f\n", Integer.parseInt(valores1[1]) * Double.parseDouble(valores1[2]) + Integer.parseInt(valores2[1]) * Double.parseDouble(valores2[2]));
    }
}